### DISCUSSION TITLE
Drop support for .NET Core 3.1

### DISCUSSION THREAD

**Discussion by ismaelhamed** [akkadotnet/akka.net#6527] [Ideas] [answered]
It'll cut test runs down, and it's out of support by Microsoft anyway.

**Comment by Aaronontheweb** [comment, marked as answer]
@ismaelhamed I think I speak for @Arkatufus as well as myself - we 100% endorse this.

**Reply by Aaronontheweb** [reply]
In many of our internal projects (i.e. Phobos, Petabridge.Cmd) we've already largely dropped .NET Core 3.1 support and are just targeting .NET Framework and .NET 6/7.

**Comment by CumpsD** [comment]
In favor as well! :)
